react-basic-practice/src/components/Modal/index.tsx

```diff
@@ -9,23 +9,34 @@ import {
 } from "@chakra-ui/react";
 import Button from "../Button";
 import Form from "../Form";
+import { ICustomer } from "@types";
 
 interface ModalProps {
   isOpen: boolean;
   onClose: () => void;
   headerText?: string;
+  viewData?: ICustomer;
 }
 
 const Modal: React.FC<ModalProps> = ({
   isOpen,
   onClose,
   headerText = "Add Customer",
 }) => {
+  const isEdit = headerText === "Edit Customer";
+  const isView = headerText === "View Customer";
+
   return (
     <ChakraModal isOpen={isOpen} onClose={onClose}>
       <ModalOverlay />
       <ModalContent maxWidth="794px" padding="0 40px">
-        <ModalHeader padding="23px 0 18px 0" borderBottom="1px solid black" fontWeight="semibold" fontSize="lg" color="mirage">
+        <ModalHeader
+          padding="23px 0 18px 0"
+          borderBottom="1px solid black"
+          fontWeight="semibold"
+          fontSize="lg"
+          color="mirage"
+        >
```

Comment:
Consider to use conditional (ternary) operator (? :)
refer: https://react.dev/learn/conditional-rendering#conditional-ternary-operator--